src/main/pages/che-7/overview/assembly_running-che-locally.adoc

```diff
@@ -69,32 +69,34 @@ Install the `chectl` command-line tool to be able to manage the Che Server appli
 
 === Installing chectl on Windows
 
-. Download link:https://github.com/che-incubator/chectl/releases/download/20190314232124/chectl-win.exe[chectl-win.exe].
+. Navigate to link:https://github.com/che-incubator/chectl/releases[chectl Releases].
 
-. Add the folder of the downloaded binary to your `PATH` environment variable.
+. Download the latest archive suitable for the target architecture.
 
-=== Installing chectl on macOS
+** Windows 64 bits: `chectl-win32-x64.tar.gz`
 
-Run the following commands in the terminal:
+** Windows 32 bits: `chectl-win32-x86.tar.gz`
 
-----
-$ RELEASE="20190314232124/chectl-macos" && \
-URL=https://github.com/che-incubator/chectl/releases/download/${RELEASE} && \
-sudo curl -sSL ${URL} -o /usr/local/bin/chectl && \
-sudo chmod +x /usr/local/bin/chectl 
-----
+. Extract the archive.
+
+. The files are extracted in a directory called `chectl`. Binaries are in the `bin` subdirectory.
 
-=== Installing chectl on Linux
+. Add the full path of the `chectl\bin` directory to your `PATH` environment variable.
 
-Run the following commands in the terminal:
+=== Installing chectl on Linux or MacOS
+
+.Prerequisites
 
+. `/usr/local/bin` in in your `$PATH`
+
+.Procedure 
+
+. Run the following commands in the terminal:
++
 ----
-$ RELEASE="20190314232124/chectl-linux" && \
-URL=https://github.com/che-incubator/chectl/releases/download/${RELEASE} && \
-sudo curl -sSL ${URL} -o /usr/local/bin/chectl && \
-sudo chmod +x /usr/local/bin/chectl 
+# bash <(curl -sL  https://www.eclipse.org/che/chectl/)
```

Comment:
FYI here is what I added also

```
To get the version from master branch instead of stable:
----
$ bash <(curl -sL  https://www.eclipse.org/che/chectl/) --channel next
----

=== Updating chectl

Chectl can be updated by running the command
----
$ chectl update
----
It will update chectl based on its current channel.

To move to a different channel, there is an optional argument
----
$ chectl update [next|stable]
----

```

Reply:
as I think it's important to say that channel can be changed and updating is also available

Reply:
Should we really document here how to switch to unreleased versions of chectl? That was previously not the case and it is beyond the scope of current PR. Happy to discuss this later.

Documenting how to update chectl is also beyond the scope of current PR, but I am more than happy to submit it later. As the script from https://www.eclipse.org/che/chectl/ is installing `chectl` into `/usr/local/bin`, `chectl update` needs certainly to be run with superuser privileges.

Reply:
-> https://github.com/eclipse/che/issues/14237